src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (sourceString.length === 0) {
+    return {};
+  }
+
+  const stringToObject = sourceString
+    .split(';')
+    .map((value) => {
+      return value.split(':').map((withoutSpace) => withoutSpace.trim());
+    })
+    .reduce((acc, value) => {
+      acc[value[0]] = value[1];
```

Comment:
The code does not handle cases where there are multiple semicolons without any styles between them. It would be nice to filter out any empty style declarations before reducing them into an object.